plugin.program.bluetoothctl/resources/lib/bluetoothctl.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
```

Comment:
the empty __init__.py file is not needed with Python3 and should be removed.  This also applies for the ones in the resources and lib folders.